src/hqdn3d.cpp

```diff
@@ -124,12 +124,38 @@ deNoise(
 
             prevPlane[row * frameWidth + col]
                 = ((resPix + ROUND_LINE) >> SHIFT_LINE) & 0xFFFF;
-            tarPlane[row * tarStride + col]
-                = (resPix + ROUND_PIXEL) >> SHIFT_PIXEL;
+            if (tarPlane)
+                tarPlane[row * tarStride + col]
+                    = (resPix + ROUND_PIXEL) >> SHIFT_PIXEL;
         }
     }
 }
 
+static void filterFrame(const VSFrameRef *srcFrame, VSFrameRef *newFrame, const bool isFirstFrame, Hqdn3dData *usrData, const VSAPI *vsapi) {
```

Comment:
Splitting long lines is preferred, but not mandatory

Reply:
I'll split it.